utils/lists.py

```diff
@@ -6,7 +6,26 @@
     "You'll be the judge", 'no... (╯°□°）╯︵ ┻━┻', 'no... baka',
     'senpai, pls no ;-;'
 ]
+i = 0
+answers = []
+questions = []
 
+with open ("/Users/sethsarchet/Desktop/QuestionList.txt") as fp:
```

Comment:
This needs changed to a relative path. Pathnames to files should not include user specific names.